libs/search/src/lib/sort-by/sort-by.component.ts

```diff
@@ -5,25 +5,22 @@ import { SearchState } from '../state/reducer'
 import { getSearchSortBy } from '../state/selectors'
 import { marker } from '@biesbjerg/ngx-translate-extract-marker'
 
-marker('last changed')
-marker('popularity')
-
 @Component({
   selector: 'search-sort-by',
   templateUrl: './sort-by.component.html',
 })
 export class SortByComponent implements OnInit {
   choices = [
     {
-      label: 'relevancy',
+      label: 'results.sortBy.relevancy',
```

Comment:
how those strings will be reconized to be extracte d?

Reply:
In this particular case, I don't think we can rely on the extractor script - at some point this will not be hardcoded and we may have a list or config for this ... but I can restore the marker and use the npm script if it makes more sense ...

Reply:
Well, I think it's better to keep the translation keys being extracted. It's a good practice to let the script update the file.


> at some point this will not be hardcoded and we may have a list or config for this

What do you mean exactly ?
If we update `en.json` manually to add foreign keys, it's the same effort to add them as markers in a dedicated file.

Reply:
Also if we use ` npm run i18n:extract`

```json
{
	"search.field.any.placeholder": "Search datasets, services and maps ...",
	"results.layout.selectOne": "Results layout",
	"search.field.sortBy": "Sort by",
	"records": "records",
	"record.action.view": "viewable",
	"record.action.download": "downloadable",
	"record.more.details": "Read more",
	"results.records.hits.found=0.help": "Suggestions: <ul class='list-disc list-inside'><li>Try other words</li><li>Specify fewer words</li></ul>",
	"results.records.hits.found": "{hits, plural, =0{No documents match the specified search.} one{} other{{hits} records found.}}",
	"search.loading": "Loading ...",
	"results.sortBy.relevancy": "Relevancy",
	"results.sortBy.dateStamp": "Last updates",
	"results.sortBy.popularity": "Popularity",
	"dropFile": ""
}
```

which looks to be incomplete. Should we then add markers for all facets stuffs ? Web component translation like `nav.back` are also missing.

Reply:
Does it work well for the datafeeder app ?

Reply:
> Does it work well for the datafeeder app ?

Not sure they are using, nobody read the manual ! :)

Reply:
> Should we then add markers for all facets stuffs ? 

IMO yes

> Web component translation like `nav.back` are also missing.

Maybe it's just the path config in the package.json script

It helps to see some translations are missing, or some key are not well formatted, I keep thinking it's a good practice but I won't force it you find it painfull to maintain.

Reply:
Let's see if it works. Markers added. Format consistent with prettier. Web component path fixed.